en/simulation/gazebo.md

```diff
@@ -367,6 +367,23 @@ gst-launch-1.0  -v udpsrc port=5600 caps='application/x-rtp, media=(string)video
 ! rtph264depay ! avdec_h264 ! videoconvert ! autovideosink fps-update-interval=1000 sync=false
 ```
 
+### Verbose Logging
+
```

Comment:
@Jaeyoung-Lim Please check - and if this logs specifically only part of gazebo (server or client) or any other very specific part, we should identify that part. If it is for general use we can probably leave as is.

Reply:
@Jaeyoung-Lim wink wink :point_up: